espnet2/enh/espnet_model.py

```diff
@@ -193,9 +193,22 @@ def forward_enhance(
                 feature_mix, flens
             )
             if additional.get("num_spk") is not None:
-                feature_pre, flens, others = self.mask_module(
-                    feature_mix, flens, bottleneck_feats, additional["num_spk"]
-                )
+                if additional.get("batch_num_spk") is not None:
```

Comment:
Can you add some comments here to describe the difference between "num_spk" and "batch_num_spk"?